lib/test.js

```diff
@@ -104,51 +112,52 @@ Test.prototype.plan = function (count) {
 };
 
 Test.prototype.run = function () {
-	this.promise = {};
+	var self = this;
+
+	this.promise = Promise.pending();
```

Comment:
The main reason for using `Promise.pending()` is to have less nesting/indentation in this function, that's it.